TransformationSystem/Client/TaskManager.py

```diff
@@ -333,6 +335,116 @@ def __init__( self, transClient = None, logger = None, submissionClient = None,
     self.destinationPlugin_o = None
 
   def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    bulkFlag = self.opsH.getValue( "Transformations/BulkSubmission", False )
```

Comment:
Why you do not want to apply bulk submission to all the job related transformations ?